ambry-server/src/integration-test/java/com.github.ambry.server/ServerHardDeleteTest.java

```diff
@@ -300,10 +300,13 @@ public void endToEndTestHardDeletes() throws Exception {
     notificationSystem.awaitBlobDeletions(blobIdList.get(4).getID());
 
     time.sleep(TimeUnit.DAYS.toMillis(1));
-    // Changes in this patch: a. New header version has 4 more bytes compared to previous b. BlobProperties increased by 1 byte
-    // c. Encryption Key Record size is 114 for an encryptionKey of size 100. EncryptionKeyRecord could be null if not applicable.
+    // Changes in this patch:
+    // a. New header version has 4 more bytes compared to previous
+    // b. BlobProperties increased by 1 byte
+    // c. Encryption Key Record size is 114 for an encryptionKey of size 100.
+    //    EncryptionKeyRecord could be null if not applicable.
     // Delta: 7 * 4 (headers for 7 records) + 1*6 (BlobProperties for 6 put records) + 114*3 = 376
-    ensureCleanupTokenCatchesUp(chosenPartition.getReplicaIds().get(0).getReplicaPath(), mockClusterMap, 198896);
+    ensureCleanupTokenCatchesUp(chosenPartition.getReplicaIds().get(0).getReplicaPath(), mockClusterMap, 198728);
```

Comment:
also change these expected offsets back to their old values

Reply:
Same here